inc/MonteCarloPredictor.h

```diff
@@ -50,7 +50,7 @@ namespace PCOE {
          *  @param    state state of system at time of prediction
          *  @param  data ProgData object, in which prediction results \re stored
          **/
-        void predict(const double tP, const std::vector<UData>& state, ProgData& data);
+        PCOE::Prediction predict(const double tP, const std::vector<UData>& state, ProgData& data);
```

Comment:
I don't think the namespace qualifier (`PCOE::`) is necessary here. Does anything fail to compile if you remove this?